.github/workflows/flow.yml

```diff
@@ -11,11 +11,12 @@ jobs:
     strategy:
       matrix:
         runner: [ubicloud-standard-16-ubuntu-2204-arm]
+        postgres-version: [15, 16, 17]
```

Comment:
we need only run a subset of tests ideally; PG-PG and some tests to ensure initial load/resync works; fine for now
13 and 14 should be here but they're pre-MERGE so will need work

Reply:
can you create an issue for now @heavycrystal ?